test_suite.py

```diff
@@ -80,26 +86,30 @@ def test_ownership3_4(child_sub, samples_sent, timeout, verbosity):
 
     return ReturnCode.RECEIVING_FROM_ONE
 
-# It checks if the Subscriber is receiving the samples in order.
-# It does <max_samples_received> iterations and at each of them it
-# will process one sample (taken from pexpect). For each sample it
-# checks if the sample received matches with the sample sent (taken
-# from <samples_sent[0]). In case if does not match it means the
-# subscriber is not receiving the samples in order, and it will
-# save the code "DATA_NOT_CORRECT", stopping the function.
-# In case it matches it will receive another sample and it will
-# continue with another iteration.
+
 def test_reliability_4(child_sub, samples_sent, timeout, verbosity):
+
+    """
+    The functions testS reliability, checking if the Subscriber receives the
```

Comment:
testS reliability? What does this refer to?
```suggestion
    The functions testS reliability check whether the Subscriber receives the
```
If is conditional, whether is for yes/no situations